src/components/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+import { MakeChange } from '../types/MakeChange';
+import { Todo } from '../types/Todo';
+import { LinksPath } from '../types/LinksPath';
+
+type Props = {
+  todos: Todo[],
+  setTodos: MakeChange,
+  amountCompletedTodos: number,
+};
+
+const filterLinks = [{
+  title: 'All',
+  to: LinksPath.All,
+}, {
+  title: 'Active',
+  to: LinksPath.Active,
+}, {
+  title: 'Completed',
+  to: LinksPath.Completed,
+}];
+
+export const Footer: FC<Props> = ({
+  todos,
+  setTodos,
+  amountCompletedTodos,
+}) => {
+  const hendlerRemoveAll = () => {
```

Comment:
Consider renaming 'hendlerRemoveAll' to 'handleRemoveAll' for better readability.